### PR TITLE
Update staging by syncing with upstream

### PR DESCRIPTION
**Pull request by DavidTheProgrammer**
## Description
This PR updates our version of the Wazimap-ng backend with the latest changes from the upstream repo. It also maintains the addition of the `display_format` field in the `ProfileKeyMetrics` model that is used internally and resolves all conflicts arising from that change.
Other changes are in regards to migrations, the PR merges some diverging leaves of root migrations following the addition of our custom field as explained above.
_**Some tests are failing**_ and this is because they are failing even on the upstream branch, I've fixed some of them that were failing due to the addition of the `display_format` field and other easy to catch errors. Considering opening a PR to push the changes upstream??
View commits for full changes after conflict resolution, including the fixing of some of the failing test files. 

## Related Issue

https://github.com/CodeForAfrica/wazimap-ng/issues/4

## How to test it locally
Run the docker compose command as shown in the `README.md` and verify it's working as expected.

### Added
- `AWS_QUERYSTRING_AUTH` to readme and updated it's location to the main settings file. Changed read mode from `os.environ.get` to `env.str` to align with upstream.

### Updated
- Location of `DISPLAY_FORMAT_CHOICES` constant to align with upstream changes of moving it to `constants.py` file.
- Resolved conflicts in multiple files resulting from the addition of the `display_format` field.

### Removed
- Additional settings files to align with upstream changes.

## Checklist

- [x]  🚀 is the code ready to be merged and go live?
- [x]  🛠 does it work (build) locally

### Pull Request

- [x]  📰 good title
- [x]  📝good description
- [x]  🔖 issue linked
- [x]  📖 changelog filled out

### Commits

- [x]  commits are clean
- [x]  commit messages are clean

### Code Quality

- [x]  🚧 no commented out code
- [x]  🖨 no unnecessary logging
- [x]  🎱 no magic numbers
- [x]  black was run locally (as part of the pre-commit hook)

### Testing

- [x]  ✅ added (appropriate) unit tests
- [x]  💢 edge cases in tests were considered
- [x]  ✅ ran tests locally & are passing - Mostly

### CODE REVIEW

**Comment by kilemensi** [comment]
👍🏽 @DavidTheProgrammer.

Have we tested this against our FE apps (`climatemappedafrica` and `pesayetu`) and all works well?

**Comment by satoru260** [comment]
> 👍🏽 @DavidTheProgrammer.
> 
> Have we tested this against our FE apps (`climatemappedafrica` and `pesayetu`) and all works well?

I tested it against climatemappedafrica but not pesayetu. Let me try it now, but from all indications I don't expect any issues.

File: docker-compose.yml [hunk 1]
```diff
@@ -1,13 +1,25 @@
...
+  DJANGO_Q_SYNC:
+  EMAIL_BACKEND: django.core.mail.backends.console.EmailBackend
+  ENVIRONMENT_NAME: Dev
+  WERKZEUG_DEBUG_PIN: "off"
+  #DEBUG_CACHE_AS_PRODUCTION: True
 
+services:
   db:
-    image: kartoza/postgis:12.0
+    image: kartoza/postgis:11.5-2.8
```
Comment (kilemensi) [inline docker-compose.yml]: ~Does this mean we're downgrading? If yes, do we know why?~

... this was cleared in the standup call.

File: heroku.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+build:
```
Comment (kilemensi) [inline heroku.yml:1]: I see a lot of Heroku configs, I'm assuming we're keeping track of these in a separate issue (that will enable us to switch to Dokku).

File: requirements.txt [hunk 3]
```diff
@@ -1,9 +1,8 @@
 # Core
 pytz==2018.9
-Django==2.2.24
-django-configurations==2.1
+Django==2.2.28
```
Comment (kilemensi) [inline requirements.txt:3]: Still on Django [`2`](https://endoflife.date/django)

**Comment by satoru260** [comment]
The issue below was created as a test branch to test this backend with local applications, this was done and the climatemapped application is working as expected. Screenshots can be found in the issue:

https://github.com/CodeForAfrica/wazimap-ng/issues/6

**Comment by satoru260** [comment]
> The issue below was created as a test branch to test this backend with local applications, this was done and the climatemapped application is working as expected. Screenshots can be found in the issue:
> 
> #6

The application has been tested against the current version of the backend _with data_ and the migrations went smoothly. To reproduce, checkout to this branch and do the following: 

1. Start the database by running `docker compose run --rm web python wait_for_postgres.py`.
2. Restore the database with a dump file of the current database.
3. Run the migrations with `docker compose run --rm web python manage.py migrate`.
4. Start the rest of the stack with `docker compose up -d`.
5. Verify that all data is fine and any UI applications can pull data accordingly.

**Review by kilemensi — APPROVED** [review]
Code LGTM 🚀

**Review by thepsalmist — APPROVED** [review]
LGTM
